src/main/app/case/answers/possibleAnswers.ts

```diff
@@ -44,7 +45,19 @@ export const omitUnreachableAnswers = (caseState: Partial<Case>, steps: Step[]):
   pick(caseState, getAllPossibleAnswers(caseState, steps));
 
 export const getUnreachableAnswersAsNull = (userCase: Partial<Case>): Partial<Case> => {
-  const possibleAnswers = getAllPossibleAnswers(userCase, stepsWithContent);
+  const possibleAnswers = [
+    ...getAllPossibleAnswers(userCase, stepsWithContent),
+    ...(userCase.applicationType === ApplicationType.JOINT_APPLICATION
+      ? getAllPossibleAnswers(
+          userCase,
+          stepsWithContent.slice(
```

Comment:
```suggestion
      getAllPossibleAnswers(userCase, stepsWithContentApplicant2),
```

Reply:
would this not work?

Reply:
As discussed, we cannot make this change until Seb's branch is merged in at a later date. This can be done as some refactoring once this happens.